pydrive2/files.py

```diff
@@ -687,6 +685,22 @@ def _WrapRequest(self, request):
         """
         if self.http:
             request.http = self.http
+        request = self._AddResourceKeyHeaders(request)
+        return request
+
+    def _AddResourceKeyHeaders(self, request):
+        """Add resourceKey headers to request if file is secured with resourceKey and
+        its available (from a list for example).
+
+        :param request: request to add headers to.
+        :type request: googleapiclient.http.HttpRequest
+        """
+        file_id = self.metadata.get("id") or self.get("id")
+        resourceKey = self.get("resourceKey")
+        if file_id and resourceKey:
```

Comment:
one minor ask, just to be 100% safe can we do something like this please:

```
if resourceKey:
   file_id = ....
   if file_id:
      ....
```

Reply:
Done

Reply:
thanks!